lib/modules/uscore_v3.1.1/us_core_diagnosticreport_note_sequence.rb

```diff
@@ -729,6 +729,33 @@ def patient_ids
           validate_reference_resolutions(resource, validated_resources, max_resolutions) if validated_resources.length < max_resolutions
         end
       end
+
+      test 'Every attachment within DiagnosticReport resources can be read.' do
+        metadata do
+          id '14'
+          link 'https://www.hl7.org/fhir/datatypes.html#attachment'
+          description %(
+
+            This test will attempt to read an attachment in the resources from the first search.
+            The test will fail if Inferno fails to read any of those references.
+
+          )
+          versions :r4
+        end
+
+        skip_if_not_found(resource_type: 'DiagnosticReport', delayed: false)
+
+        found_one_attachment = false
+
+        @diagnostic_report_ary&.values&.flatten&.each do |resource|
+          if validate_attachment_resolutions(resource, ['presentedForm'])
+            found_one_attachment = true
+            break
```

Comment:
The description says that this test attempts to read all attachments and fails if it is unable to read any, but this code indicates that it stops reading attachments once it successfully reads one, and as long as a single attachment can be read, all other can fail.

Reply:
Will update the description to match the new testing logic.

Reply:
Is only requiring one attachment to work the behavior we want?

Reply:
Yes. One is enough to demonstrate the server "support" attachment download